summoner-cli/src/Summoner/Template/GitHub.hs

```diff
@@ -107,6 +109,76 @@ gitHubFiles Settings{..} = concat
         then ""
         else unlines ("\n# User specific" : settingsGitignore)
 
+    ghActionsYml :: Text
+    ghActionsYml = [text|
+        name: Haskell CI
+
+        on:
+          # Trigger the workflow on push or pull request,
+          # but only for the master branch
+          push:
+            branches:
+              - master
+          pull_request:
+
+        jobs:
+          build:
+            name: ghc ${{ matrix.ghc }}
+            runs-on: ubuntu-16.04
```

Comment:
Oh, also, maybe we should use `18.04`?

Reply:
I’m honestly not sure. I don’t know what best accepted practice for Actions here is.

Reply:
Let's leave it for now. We can change it after testing Summoner more if needed.